ios/Sources/MediaFeature/MediaScreen.swift

```diff
@@ -29,6 +29,7 @@ public struct MediaScreen: View {
         NavigationView {
             WithViewStore(store) { viewStore in
                 ZStack {
+                    AssetColor.Background.primary.color.ignoresSafeArea()
```

Comment:
This is actually broken by the animation of the `ZStack`.
But I found out that the animation can cause other problems, so I remove it in #571.